### PR TITLE
Add company info component for internship post card

### LINKED ISSUE #57: [TASK] Add Internship Company Info component
## Description
Create `InternshipCompanyInfo.jsx` component to include the company logo, Name, title, and location
As a user looking at an internship card I want to be able to see the company logo, name, title, and location in the top left corner

Image

## Related
#25

### PR DESCRIPTION
**Pull request by aqvilala**
#### What is the purpose of your change?
- Fixes #57 

#### What areas of the application does this change impact? (changes API, folder structure, etc)
- Adding another page in the components folder
- Updating App.jsx to include additional colours (can be removed? Jenna mentioned adding it in a previous meeting but I'm not sure if we've changed our plan for organizing styling colours)

#### What features/fixes are deferred/not handled in this PR?
- N/A

## Checklist
- [ ] Adds unit tests
- [ ] Has a work item linked to the PR

### CODE REVIEW

**Comment by jwu910** [comment]
can we take a look at resolving the merge conflicts and addressing the CI checks before we review? @aqvilala

**Comment by jwu910** [comment]
Is there any support we can provide for this @aqvilala ?

File: client/src/App.jsx [hunk 1]
```diff
@@ -24,6 +25,39 @@ const BRAND_COLORS = {
 const theme = createTheme({
   palette: {
     background: { paper: BRAND_COLORS.MILK_TEA, default: "#FAF0E7" },
+    brown: {
+      lightest: "#FAF0E7",
```
Comment (michachan) [inline client/src/App.jsx:29]: Could we use a [color:number] pattern?

brown: {
'100': '#FAF0E7' etc
}

File: client/src/components/InternshipCompanyInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <>
+      <Box
+        display="flex"
+        maxWidth={false}
+        sx={{
+          maxWidth: "fit-content",
+        }}
+      >
+        <Avatar sx={{ width: 69, height: 69 }} variant="rounded"></Avatar>
```
Comment (michachan) [inline client/src/components/InternshipCompanyInfo.jsx:16]: prefer self-closing tag

File: client/src/components/InternshipCompanyInfo.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { Box, Avatar, Typography, useTheme } from "@mui/material";
+
+const InternshipCompanyInfo = (props) => {
+  let theme = useTheme();
```
Comment (michachan) [inline client/src/components/InternshipCompanyInfo.jsx:5]: prefer const for variables that aren't re-defined

**Review by michachan — APPROVED** [review]

**Comment by jwu910** [comment]
@aqvilala Let us know if you need any help with this one as well, thanks

**Comment by aqvilala** [comment]
Hello, sorry for the delay! Here's what it looks like on my local: 
image

I've also updated the PR and resolved the conflicts.

File: client/src/components/InternshipCompanyInfo.jsx [hunk 4]
```diff
@@ -13,21 +13,42 @@ const InternshipCompanyInfo = (props) => {
...
+            height: "69px",
+            marginTop: "0.25rem",
+          }}
+        />
+        <Box sx={{ paddingLeft: "1rem", marginTop: "-0.25rem" }}>
           <Typography
             variant="h5"
             sx={{
-              color: theme.palette.info.main,
+              color: "blue.100",
```
[the remark this answers is not in the document]
Comment (aqvilala) [reply client/src/components/InternshipCompanyInfo.jsx:28]: There's no longer a theme.palette.info with Jenna's new changes to the theme.js file so I had to switch it over to blue.100 instead.

File: client/src/components/InternshipCompanyInfo.jsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+          variant="rounded"
+          sx={{
+            width: "69px",
+            height: "69px",
+            marginTop: "0.25rem",
+          }}
+        />
+        <Box sx={{ paddingLeft: "1rem", marginTop: "-0.25rem" }}>
+          <Typography
+            variant="h5"
```
[the remark this answers is not in the document]
Comment (aqvilala) [reply client/src/components/InternshipCompanyInfo.jsx:26]: These are the existing prestyled variants: 
image

Should I create a new one in theme.js to match the one I've used for the company title?

[the remark this answers is not in the document]
Comment (jwu910) [reply client/src/components/InternshipCompanyInfo.jsx:26]: Ah i see, so we're not adding uniform styles to these, no biggie. we can leave it for now, and if we need to clean it up later, we can

**Comment by jwu910** [comment]
@all-contributors please add @aqvilala for code

**Pull request merged**